google/cloud/pubsub/samples/topic_admin_samples.cc

```diff
@@ -219,9 +219,9 @@ void CreateTopicWithCloudStorageIngestion(
   namespace pubsub = ::google::cloud::pubsub;
   namespace pubsub_admin = ::google::cloud::pubsub_admin;
   [](pubsub_admin::TopicAdminClient client, std::string project_id,
-     std::string topic_id, std::string bucket, std::string input_format,
+     std::string topic_id, std::string bucket, std::string const& input_format,
```

Comment:
Same question as above.

Reply:
We should `std::move` it along with some of the other strings. Fixed.